doc/en/customize.rst

```diff
@@ -20,7 +20,9 @@ which were registered by installed plugins.
 Initialization: determining rootdir and inifile
 -----------------------------------------------
 
-pytest determines a ``rootdir`` for each test run which depends on
+Pytest defines ``rootdir`` parameter which is determined for each test run.
+It should't be consufed with ``--rootdir`` command-line argument (please see `https://github.com/pytest-dev/pytest/issues/6359` for example).
```

Comment:
```suggestion
It should't be confused with ``--rootdir`` command-line argument (please see `https://github.com/pytest-dev/pytest/issues/6359` for example).
```

Reply:
> It should't be confused with ``--rootdir`` command-line argument

That sounds wrong: `--rootdir` sets `rootdir` (manually).

You are more likely wanting to explain:

> Unfortunately I agree that the current interaction between rootdir and ini files is far from clear, and someone just looking at --rootdir would assume that it should point to the root of your repository/project, and from there on pytest would find pytest.ini files and start collection, but this is currently not the case.

Reply:
Oops! Wanted to write exactly that, but in shortenned manner :/ Maybe the better option would be re-writing sentence of @nicoddemus like "It shouldn't be confused with the place, from where pytest would find pytest.ini files and start collection (for example,...)"?